### ISSUE TITLE
Some issues about the function "get_adv2()" in nn.py

### ISSUE THREAD

**Issue by JialeDeng** [Dongdongshe/neuzz#19]
Hi!
I am reading your paper and code recently, they're really good. But I have some difficulties understanding the following code in get_adv2() in nn.py:
`adv_list = []
    loss = layer_list[-2][1].output[:, f]
    grads = K.gradients(loss, model.input)[0]
    iterate = K.function([model.input], [loss, grads])`
What does the 'loss' mean here? Does it means the specific loss of the f^th output_neuron?
